int32 id

float32 x
float32 y

# enum: HOVER, CRUISE, AUTO
int16 move_mode

# enum: ABS, REL
int16 coord_system

float32 fwd_kp
float32 fwd_ki
float32 fwd_kd

float32 side_kp
float32 side_ki
float32 side_kd

float32 timeout
